contracts/external/compound/CTokenInterface.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity 0.6.4;
+pragma solidity 0.6.12;
```

Comment:
I think interfaces should be defined with a compiler range, so that they may be imported into 3rd-party projects